scss/os/_overrides_jqueryui.scss

```diff
@@ -15,7 +15,7 @@
   }
 
   &.ui-widget-content {
-    border: $border-width solid $border-color;
+    border: 0;
```

Comment:
The spinner elements had a border around them that made them even taller than the rest of the inputs in the filter rows (which are already too tall).

Ever since BS4, the inputs in the filter tree look terrible. We originally had them set to a shorter height, which looked bad, now they're too tall and clip out. I played with upping the row height, but didn't like that either because it chewed up a ton more vertical space. 

I will look at making this more specific or taking it out.